bindings/python/tests/test_session.py

```diff
@@ -0,0 +1,34 @@
+# test cases for Session wrapper object
+#
+# test for get_book may belong in test_book but it makes sense here
+# to see if get_current_session works
+# test for app_utils on the other hand could go to a subfolder of 
+# /libgnucash/app-utils
+#
+# @date 2020-04-03
+# @author Christoph Holtermann <mail@c-holtermann.net>
+
+from unittest import TestCase, main
+
+from gnucash import Session
+
+class TestSession( TestCase ):
```

Comment:
Whitespace around parentheses is [unpythonic](https://www.python.org/dev/peps/pep-0008/#whitespace-in-expressions-and-statements).